cmake/release_dependencies.cmake

```diff
@@ -45,6 +45,11 @@ FetchContent_Declare( Log
     GIT_TAG         52962b7796afe37ef1d8f7edb4bf9ecb1b868d15
     )
 
+FetchContent_Declare( pybind11
+    GIT_REPOSITORY  https://github.com/pybind/pybind11
+    GIT_TAG         f1abf5d9159b805674197f6bc443592e631c9130 # tag: v2.6.1
+    )
+
 FetchContent_Declare( range-v3-adapter
```

Comment:
Is there still a reason for the adapter?

Reply:
That's going to be another pull request (see #32)